### PR TITLE
Minio resource initialization

### PR DESCRIPTION
**Pull request by Mun1z**
Implementation of minio bucket configuration support on resource initialization script.

image

### CODE REVIEW

**Review by gilMars — APPROVED** [review]

File: docs/resources_initialization.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If you are setting up a bucket of azure type:
 
 - [Azure CLI](https://docs.microsoft.com/pt-br/cli/azure/): The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.
 
+If you are setting up a bucket of minio type:
+
+- [MinIO](https://min.io/): That said, you can configure it in the same way as described above by placing Access Key and Secret Access Key of your MinIO bucket.
```
Comment (lucasncabral) [inline docs/resources_initialization.md:33]: To set up a bucket on the MinIO the user must also have the AWS CLI installed. Also, the user needs to have MinIO running on the desired endpoint, right? I believe this message is not clear.

File: docs/resources_initialization.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ If you are setting up a bucket of azure type:
 
 - [Azure CLI](https://docs.microsoft.com/pt-br/cli/azure/): The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.
 
+If you are setting up a bucket of minio type:
```
Comment (lucasncabral) [inline docs/resources_initialization.md:31]:
```suggestion
If you are setting up a bucket of MinIO type:
```

File: scripts/resources_initialization/resources_initialization.sh [hunk 3]
```diff
@@ -42,19 +42,33 @@ create_new_github_repository()
 
 create_new_bucket()
 {
-   read -p "What type of storage do you want to configure? [s3h, azureblobh]: " STORE_TYPE
+   read -p "What type of storage do you want to configure? [s3h, azureblobh, minio]: " STORE_TYPE
    read -p "What name do you want to give to your bucket? " BUCKET_NAME
-   echo "  ${STORE_TYPE}:" >> config.yaml
-   echo "    ${BUCKET_NAME}:" >> config.yaml
```
[the remark this answers is not in the document]
Comment (Mun1z) [reply scripts/resources_initialization/resources_initialization.sh:48]: When the selected storage is MinIO, unlike the other two, it inserts the 's3h' in config.yaml, so it was not possible to reuse the same piece of code between the different types, any suggestions?

File: docs/resources_initialization.md (hunk 2 above)
Comment (Mun1z) [reply docs/resources_initialization.md]: Done on commit: https://github.com/HPInc/ml-git/pull/112/commits/45404c670c4389a7a5ad919044f21bbd0fd84eda

File: docs/resources_initialization.md (hunk 1 above)
Comment (Mun1z) [reply docs/resources_initialization.md]: Done on commit: https://github.com/HPInc/ml-git/pull/112/commits/45404c670c4389a7a5ad919044f21bbd0fd84eda

File: scripts/resources_initialization/resources_initialization.sh [hunk 4]
```diff
@@ -42,19 +42,33 @@ create_new_github_repository()
...
-
+   read -p "What endpoint url connection to your bucket? " ENDPOINT
```
[the remark this answers is not in the document]
Comment (Mun1z) [reply scripts/resources_initialization/resources_initialization.sh]: Done on commit: https://github.com/HPInc/ml-git/pull/112/commits/45404c670c4389a7a5ad919044f21bbd0fd84eda

**Review by andersonpablosilva — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

File: docs/resources_initialization.md [hunk 5]
```diff
@@ -28,6 +28,10 @@ If you are setting up a bucket of azure type:
 
 - [Azure CLI](https://docs.microsoft.com/pt-br/cli/azure/): The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.
 
+If you are setting up a bucket of MinIO type:
+
+- [MinIO](https://min.io/): In addition to having MinIO configured and running you will also need the AWS Command Line Interface (CLI) to perform with it.
```
Comment (gpretto) [inline docs/resources_initialization.md:33]:
```suggestion
- [MinIO](https://min.io/): In addition to having MinIO configured and running, you will also need the AWS Command Line Interface (CLI) to perform with it.
```

**Review by gpretto — APPROVED** [review]

**Review by diego-pedroso — APPROVED** [review]

**Pull request merged**